.github/workflows/maven.yaml

```diff
@@ -37,6 +37,9 @@ jobs:
       - name: build with skip tests and skip images
         run: ./mvnw -T 1C -s .settings.xml clean install -U -DskipTests -Dskip.build.image=true
 
+      - name: find all tests
```

Comment:
Is it necessary to run this separately?  Can we combine it with the steps above where we skip running the tests and but find a list of tests we need to run?